### ISSUE TITLE
Database Tables and Fields setup

### ISSUE THREAD

**Issue by TomDra** [jbb1999/CSGOtradeupgenerator#1]
I am trying to run the code with a connected SQL Database.
I have gotten all the required table names, but I'm unsure of what fields are required,
The tables are ["tradeup", "milspec", "restricted", "industrial", "covert", "consumer", "classified", "error"].
I am looking through the code and it's quite hard to follow.

Any assistance would be great.

**Comment by jbb1999** [comment]
I am working on making the code a bit simpler and have uploaded a .sql file that you can import and will contain all the structures and stuff you need!

**Comment by jbb1999** [comment]
Im working on making the code less manual!
